### ISSUE TITLE
Create architecture vision

### ISSUE THREAD

**Issue by bajiat** [apinf/platform#2444]
Placeholder for sprint task, definition to be added.

**Comment by ilarimikkonen** [comment]
yes, good idea. we need to take a look at the big themes (analytics, settings, user management, payment), see what others are doing .
